scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/sop.yaml

```diff
@@ -0,0 +1,49 @@
+FileHeader:
+  library_Suffix: 'SO'
+  device_type: 'SOP'
+
+SOP-8-1EP_4.57x4.57mm_P1.27mm_EP4.57x4.45mm:
+  size_source: 'https://ww2.minicircuits.com/case_style/XX112.pdf'
+  body_size_x:
+    nominal: 4.57
+    tolerance: 0.01
+  body_size_y:
+    nominal: 4.57
+    tolerance: 0.01
+  overall_size_x:
+    nominal: 10.16
+    tolerance: 0.01
+  lead_width:
+    nominal: 0.38
+    tolerance: 0.01
+  lead_len:
+    nominal: 1.78
+    tolerance: 0.01
+  pitch: 1.27
+  num_pins_x: 0
+  num_pins_y: 4
+
+  EP_size_x:
+    nominal: 4.57
+    tolerance: 0.01
+  EP_size_y:
+    nominal: 4.45
+    tolerance: 0.01
+  # EP_paste_coverage: 0.65
+  EP_num_paste_pads: [2, 2]
+  #heel_reduction: 0.1 #for relatively large EP pads (increase clearance)
+  # EP_size_limit_x: 3.6  #for relatively large EP pads (increase clearance)
+  # EP_size_limit_y: 3.6  #for relatively large EP pads (increase clearance)
+
+  thermal_vias:
+    count: [2, 3]
+    drill: 0.51
+    # min_annular_ring: 0.15
+    paste_via_clearance: 0.1
+    EP_num_paste_pads: [2, 2]
+    # paste_between_vias: 1
+    # paste_rings_outside: 1
+    EP_paste_coverage: 0.65
```

Comment:
You might want to increase this a bit as you have paste overlapping vias. (I would suggest 0.7 to counteract solder loss by wicking)